include/charge_controller/dev/ControllerModel.hpp

```diff
@@ -0,0 +1,167 @@
+#ifndef CHARGE_CONTROLLER_CONTROLLERMODEL_HPP
+#define CHARGE_CONTROLLER_CONTROLLERMODEL_HPP
+
+#include <cstdint>
+
+#define MAX_VOLTAGE 100
+#define MAX_CURRENT 60
+
+#define DEFAULT_VOLTAGE 96
+#define DEFAULT_CURRENT 60
+
+/**
+ * Stores values that both the LCDDisplay and the ControllerUI need to access.
+ * Acts as the model in a Model-View-Controller (MVC) Structure where
+ * ControllerModel is the model,
+ * LCDDisplay is the display, and
+ * ControllerUI is the controller
+ */
+
+class ControllerModel {
+
+public:
+    /**
+     * default (and only) initializer for the ControllerModel class.
```

Comment:
```suggestion
     * Default (and only) initializer for the ControllerModel class.
```
This is very nitpicky, but I'd like comment lines to start with capitals